### ISSUE TITLE
External audit to smart contract code

### ISSUE THREAD

**Issue by 0xjona** [protocol/retriev#7]

**Comment by irenegia** [comment]
Step 1: finding the team and planning how to handle collaboration

**Comment by 0xjona** [comment]
Yes please @irenegia would be awesome if you could put me through a team or internal reference people; otherwise we move differently

**Comment by irenegia** [comment]
Update: waiting to sign the contract with Kudelski for a ~3weeks audit effort
